Jenkinsfile

```diff
@@ -202,15 +202,15 @@ pipeline {
                     /opt/ort/bin/set_gradle_proxy.sh
 
                     if [ "$ALLOW_DYNAMIC_VERSIONS" = "true" ]; then
-                        ALLOW_DYNAMIC_VERSIONS_PARAM="--allow-dynamic-versions"
+                        ALLOW_DYNAMIC_VERSIONS_OPTION="--allow-dynamic-versions"
```

Comment:
commit msg: These are indeed options. But I believe clikt terminology does not really matter here as clikt is an implementation detail. Also because the terminology param vs option I believe exists also outside the clikt domain.

Reply:
I believe the definition of "parameter" vs "option" becomes fuzzy in the context of command line arguments. Even [JCommander mixes these](https://jcommander.org/#_types_of_options) as the annotation is called `@Parameter` but the documentation section is titled "Types of options".

It just felt odd to call these parameters now that clikt clearly calls them options. I agree it's an implementation detail the user probably does not care about, but I wanted to openly document what was *my* personal rationale for the change.